.editorconfig

```diff
@@ -1,12 +0,0 @@
-root = true
```

Comment:
Please don't remove the files of the root

Reply:
Sorry, all root files added